Formula/xplanet.rb

```diff
@@ -44,6 +45,13 @@ class Xplanet < Formula
     sha256 "3f95ba8d5886703afffdd61ac2a0cd147f8d659650e291979f26130d81b18433"
   end
 
+  # Fix compilation with giflib 5
+  # http://xplanet.sourceforge.net/FUDforum2/index.php?t=msg&th=592
```

Comment:
Can this be updated to?:
`https://xplanet.sourceforge.io/FUDforum2/index.php?t=msg&th=592`